public/util/tagManagerApi.js

```diff
@@ -57,9 +85,10 @@ export default {
   },
 
   createTag: (tag) => {
+    const clean = cleanTag(tag);
```

Comment:
I'll move it into an action because I agree React should see exactly what we're sending.

Should it be in the updateTag action though? That would mean we need to make 100% sure we always call update prior to calling save. I'm not sure if I'm missing some logic here but that seems like a bit of a manual process.

Create and save both return the tag back to React so just calling cleanTag prior to dispatching the save request should have a similar effect without requiring the programmer ensure update is called prior to saving.